R/disease_progression.R

```diff
@@ -84,3 +84,49 @@ update_to_asymptomatic_infection <- function(
     )
   }
 }
+
+#' @title Sub-patent progression for vivax model
+#' @description
+#' randomly selects individuals to clear sub-patent infections based on impact of 
+#' individual immunity to detectability
+#'
+#' @param state the human state variable
+#' @param variables the available human variables
+#' @param parameters model parameters
+#' @noRd
+create_subpatent_progression_process <- function(
```

Comment:
Would be nice for the inputs here to look more like the existing `create_progression_process`, for example I think you can have `from_state` and `to_state` arguments?